components/Api/Stability/index.module.scss

```diff
@@ -0,0 +1,36 @@
+.stability {
+  border-radius: 4px;
+  color: #fff;
+  line-height: 1.5;
+  margin: 0 0 1rem;
+  padding: 1rem;
+
+  p {
+    display: inline;
+    margin: 0;
+  }
+
+  a {
+    color: #fff;
+
+    code {
+      color: #fff !important;
+    }
+  }
+
+  &Level0 {
```

Comment:
Can the letter L be lowercase on all these "levels"?

Reply:
i will take a look